From: Lucas Kanashiro <lucas.kanashiro@canonical.com>
Date: Wed, 26 Feb 2020 17:14:24 -0300
Subject: Do not load bundler during tests

This will avoid bundler trying to load sdoc.gemspec, which would try to use
git to compile a list of files in the gem.

This patch is Debian-specific.

Author: Antonio Terceiro <terceiro@debian.org>
Forwarded: not-needed
---
 spec/spec_helper.rb | 2 --
 1 file changed, 2 deletions(-)

diff --git a/spec/spec_helper.rb b/spec/spec_helper.rb
index a7bcaba..b0fa0c9 100644
--- a/spec/spec_helper.rb
+++ b/spec/spec_helper.rb
@@ -1,5 +1,3 @@
-require 'bundler/setup'
-
 require 'sdoc'
 
 require 'minitest/autorun'
